### ISSUE TITLE
Not loading eeprom

### ISSUE THREAD

**Issue by storemanuk** [amsbr/OctoPrint-EEPROM-Marlin#9]
I've installed the plugin and see it listed with other plugins.

with printer connected I click the plugin from the left navbar then click on Load EEPROM but it doesn't list any of the eeprom settings. Just a blank page apart from a bit of text at the top and the two buttons
![marlin editor](https://user-images.githubusercontent.com/31435380/31390023-af8fefce-adca-11e7-839e-ecf3e086280c.JPG)

**Comment by amsbr** [comment]
Hi @storemanuk 

The setting #define EEPROM_SETTINGS is enabled on your firmware?

**Comment by storemanuk** [comment]
Duh! My apologies,

M503 was disabled in the firmware.
I recently uploaded Marlin 1.1.6 using a predefined configuration.h for the CR-10 but this had been disabled to allow other functions to fit in the program memory. I only discovered this last night and changed it. I'd rather have this feature than a boot screen so the boot screen went to give the room required for this feature. My newbie mistake.
